experimental/framework/react-inputs/src/CollaborativeInput.tsx

```diff
@@ -57,9 +57,9 @@ export class CollaborativeInput
     public componentDidMount() {
         // Sets an event listener so we can update our state as the value changes
         this.props.sharedString.on("sequenceDelta", (ev: SequenceDeltaEvent) => {
-            if (!ev.isLocal) {
+            // if (!ev.isLocal) {
```

Comment:
Why was this change necessary? If the event is local, then I believe we should already be updating our local state value.

Reply:
We don't render the change to the screen without removing this `isLocal`. I forget what the orignal logic was for putting this in -- @ChumpChief I think you mentioned it in our conversation last week? Do you remember?

Reply:
Filtering out local events makes sense if the CollaborativeInput is the only one editing the SharedString.  But in this demo, the SharedString is edited by both the CollaborativeInput as well as programmatically, breaking that assumption.

I don't remember if this filtering was just to minimize re-renders vs. if there was a functional aspect to it.  Probably make sure to try a few combinations of making edits in multiple tabs to verify there's no unintended functional change.